### ISSUE TITLE
Retinanet Pretrained models

### ISSUE THREAD

**Issue by Darshan2701** [facebookresearch/maskrcnn-benchmark#993]
## ❓ Questions and Help
Hi,

I wanted to train retinanet 50 FPN with 2 classes. I downloaded the pre-trained models from #102 and chopped off cls_logits variables to fit my classes. Started **train_net.py** and it did not train at all, rather it directly went ahead and did the testing part on my test dataset. 
Could you please let me know if I am missing some procedure.

**Comment by Darshan2701** [comment]
I think i need to delete the optimizer and scheduler dict, if I am not wrong before using the pre-trained model.

**Comment by Darshan2701** [comment]
Update: This did work on my dataset but the inference speed is slower than the resnet50 FPN models! 
Any pointers as to why this is happening?

**Comment by tangze5258** [comment]
> I think i need to delete the optimizer and scheduler dict, if I am not wrong before using the pre-trained model.

@Darshan2701 hi，i have same prolerms like you. i want to retrain my new datasets based on pretrained model_final.pth.And i delete  optimizer， scheduler and iteration dict too, it did work when  run train_net.py, but finally, has no new model_final.pth file generated,what's wrong ? thx....

**Comment by Darshan2701** [comment]
@tangze5258 Were you able to solve this?
Sorry for the delayed ask but.

**Comment by Darshan2701** [comment]
@tangze5258 I think u need to save the new model_final.pth in a different folder than the one used already as a new pre-trained model.
